deploy/olm-catalog/service-telemetry-operator/1.1.0/service-telemetry-operator.v1.1.0.clusterserviceversion.yaml

```diff
@@ -60,36 +60,48 @@ spec:
         version: v1
       specDescriptors:
       - description: Whether to enable collection and storage components for metrics
-        displayName: Metrics Enabled
-        path: metricsEnabled
+        displayName: Metrics
```

Comment:
The pattern in the specDescriptors changes a bit (not much) which leverages the natural grouping control noted here: https://github.com/openshift/console/blob/master/frontend/packages/operator-lifecycle-manager/src/components/descriptors/reference/reference.md#fieldgroup

It makes the UI look a bit nicer too, and allows us to group the top-level API abstractions a bit easier.

![image](https://user-images.githubusercontent.com/768751/91587783-3d50f900-e925-11ea-9ddf-a6e05917aff6.png)

Reply:
I've started down trying to figure out how to make this work. I have a feeling this isn't working as documented (but will in the future?)